mailpoet/tests/integration/Cron/CronHelperTest.php

```diff
@@ -159,11 +159,11 @@ public function testItDetectsUnknownStateOfTheDaemon() {
         'run_start' => null,
       ],
       [
-        'run_access' => $time - 4,
+        'run_access' => $time - 0.00001,
```

Comment:
For clarity, do you think it would be good to change the `4` on L146 as well (in `testItDetectsAccessibleDaemon`)? I'm not sure I fully "get" these tests anyway so maybe it makes sense to leave that one as is.